com.unity.render-pipelines.universal/Runtime/Passes/GBufferPass.cs

```diff
@@ -112,20 +112,17 @@ public override void Configure(CommandBuffer cmd, RenderTextureDescriptor camera
 
         public override void Execute(ScriptableRenderContext context, ref RenderingData renderingData)
         {
-            CommandBuffer gbufferCommands = CommandBufferPool.Get();
-            using (new ProfilingScope(gbufferCommands, m_ProfilingSampler))
+            var cmd = renderingData.commandBuffer;
+            using (new ProfilingScope(cmd, m_ProfilingSampler))
             {
-                context.ExecuteCommandBuffer(gbufferCommands);
-                gbufferCommands.Clear();
```

Comment:
Same here.

Reply:
I don't even understand the old code.
1. We get a new buffer.
2. Add a profiling scope begin.
3. Copy the buffer (A) to context. (ctx: [A)
4. Clear the current buffer
---
5. Do stuff. Potentially a new cmd buf (B)
6. Copy the buffer(B) to context (ctx: [AB)
7. Clear local
---
8. Add profiling scope end to buffer C
9. ???? Execute/Submit somewhere possibly (ctx: [ABC)
10. ???? Might break? What if someone does a new buffer and executes that before this one.

Is this manually trying fix/manage profiling scopes?

Isn't this what we're trying to fix?
Just put everything in single buffer. The scope should work as it should automatically.